<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit_calc</class>
 <widget class="QWidget" name="Deposit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBox_capital">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>167</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Капитализация вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_calc_proc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_calc_nalog">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>38</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Налог</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_calc_sum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада к кноцу срока</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_period">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Дней</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Месяцев</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Лет</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_proc">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>12</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_proc_nalog">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>12</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_period_v">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>155</width>
     <height>32</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Каждый день</string>
   </property>
   <item>
    <property name="text">
     <string>Каждый день</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Каждую неделю</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в пол года</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в год</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>В конце срока</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_period_v">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>154</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Переиодичность выплат</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_proc_t">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nalog">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Налоговая ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_period">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_list_in">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>126</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Список пополнений</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_list_is">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>94</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Список снятий</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calc">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Вычислить</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_calc_proc">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_calc_nalog">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_calc_sum">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>520</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_list_add">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать список пополнений</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_list_with">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>219</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать список снятий</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>dd/M/yyyy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>139</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата открытия вклада</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_is">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>184</width>
     <height>32</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Russia"/>
   </property>
   <item>
    <property name="text">
     <string>Раз в день</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в неделю</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в пол года</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в год</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Разовое пополнение</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_is_delete">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_is_add">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_is">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>26</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сум</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string> Ок</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_is">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>dd/M/yyyy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_with_add">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_with">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>dd/M/yyyy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_with_delete">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_with">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>26</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сум</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_with">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>184</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Раз в день</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в неделю</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в пол года</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в год</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Разовое пополнение</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="listWidget_is">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>541</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_sum">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_period">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_proc">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_with">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>55</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_is">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>55</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit_nalog">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_with">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>541</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
